### ISSUE TITLE
Inline galleries do not work

### ISSUE THREAD

**Issue by janezturk** [NotionX/react-notion-x#226]
Thank you very much for awesome library!

I think Notion changed something regarding inline galleries. 
Looks like NotionClient is not returning inline gallery as a part of the page.

Here is my example page:
https://tall-flock-373.notion.site/Problem-with-gallery-page-8a4e3d43f7f54323a2d4aca5647a6b5a

**Comment by ntcho** [comment]
It seems the recent update of inline database styles are causing this problem. 🙁

**Comment by Dineshs91** [comment]
I have noticed the same.

**Comment by ntcho** [comment]
`CollectionViewGallery`, `CollectionViewList`, `CollectionViewTable` are affected by this change.

The problem seems to be originated from the modification of the props being passed to these objects:

<table>
<tr><td>Previous props</td><td>Current props</td></tr>
<tr>
<td>

```js
{
  collection: {
    id: 'ba828d5c-47bd-4c26-...',
    ...
  },
  collectionView: {
    id: '0981a186-f238-44ac-...',
    ...
  },
  collectionData: {
    type: 'results',
    collection_group_results: {
      blockIds: [
        'c6bcd16d-ff46-4b7c-...',
      ],
    },
    hasMore: false
  },
  padding: 96,
  width: 1024
}
```

</td>
<td>

```js
{
  collection: {
    id: 'ba828d5c-47bd-4c26-...',
    ...
  },
  collectionView: {
    id: '0981a186-f238-44ac-...',
    ...
  },
  collectionData: {
    type: 'results',
    blockIds: [
      'c6bcd16d-ff46-4b7c-...',
    ],
    hasMore: false
  },
  padding: 96,
  width: 1024
}
```

</td>
</tr>
</table>

### Quick fix

Replace all `collectionData['collection_group_results'].blockId` to `collectionData.blockId` in JS files located in `/node_modules/react-notion-x/build/ejs` and `/node_modules/react-notion-x/build/esm` directiory

**Comment by transitive-bullshit** [comment]
This should be fixed in `v6.0.7`
